hiob-ludolf-centre-for-ethiopian-studies.csl

```diff
@@ -177,13 +177,13 @@
       <if variable="event">
         <choose>
           <if variable="genre" match="none">
-            <text term="presented at" text-case="capitalize-first" suffix=" "/>
+            <text term="presented at" text-case="lowercase" suffix=" "/>
```

Comment:
@eu-genia, with very few exceptions, English terms are defined in lowercase, so `text-case="lowercase"` is usually redundant and can be omitted.